pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "json-schema"
+version = "23.06.1"
```

Comment:
We will be anchoring the version name to our latest release, something that we do in other places already.

Reply:
I don't have strong feeling about about fixing versions in this particular case, where the software in secondary to the data.

Reply:
Ah, I was just describing. No strong feeling against it either